modules/notebook-images-for-data-scientists.adoc

```diff
@@ -52,6 +52,11 @@ endif::[]
 | HabanaAI
 | The HabanaAI notebook image optimizes high-performance deep learning (DL) with Habana Gaudi devices. Habana Gaudi devices accelerate DL training workloads and maximize training throughput and efficiency.
 
+| code-server
+| With the code-server notebook image, you can customize your notebook environment to meet your needs using a variety of extensions to add new languages, themes, debuggers, and connect to additional services. Enhance the efficiency of your data science work with syntax highlighting, auto-indentation, and bracket matching, as well as an automatic task runner for seamless automation. See link:https://github.com/coder/code-server[code-server in GitHub] for more information. +
```

Comment:
Redundant "+" symbol at the end of paragraph.

Reply:
Actually, not sure about this without seeing the output, so ignore if needed.

Reply:
This is to set the following note on a new line.